src/Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions/IConnectionInformation.cs

```diff
@@ -12,9 +12,10 @@ public interface IConnectionInformation
         IPEndPoint LocalEndPoint { get; }
 
         PipeFactory PipeFactory { get; }
+        IScheduler InputReaderScheduler { get; }
```

Comment:
Right now it's pretty hidden on purpose https://github.com/aspnet/KestrelHttpServer/blob/dev/src/Microsoft.AspNetCore.Server.Kestrel.Core/Internal/InternalKestrelServerOptions.cs#L10.